src/main/java/com/streamr/client/StreamrRESTClient.java

```diff
@@ -131,4 +143,13 @@ public UserInfo getUserInfo() throws IOException {
         HttpUrl url = HttpUrl.parse(options.getRestApiUrl() + "/streams/" + streamId + "/publishers");
         return get(url, publishersJsonAdapter).getAddresses();
     }
+
+    public void logout() throws IOException {
+        HttpUrl url = HttpUrl.parse(options.getRestApiUrl() + "/logout");
+        Request.Builder builder = new Request.Builder().url(url)
```

Comment:
Why not use the `post` utility function here? Is it because you don't want the retry logic in this case? Maybe the utility function could have an optional `boolean retryIfSessionExpired` parameter to control whether to retry or not, to enable this function to reuse the `post` function?